vividus/src/main/java/org/vividus/report/MetadataLogger.java

```diff
@@ -55,16 +68,38 @@ public static void logEnvironmentMetadata()
             String propertyFormat = "   %-" + maxKeyLength + "s %s%n";
             try (Formatter message = new Formatter())
             {
-                message.format("%n");
+                message.format(NEW_LINE);
                 EnvironmentConfigurer.ENVIRONMENT_CONFIGURATION.forEach((category, properties) -> {
-                    message.format("%s%n %s:%n", horizontalRule, category.getCategoryName());
+                    message.format(CATEGORY_FORMAT, HYPHEN.repeat(HORIZONTAL_RULE_LENGTH), category.getCategoryName());
```

Comment:
move `HYPHEN.repeat(HORIZONTAL_RULE_LENGTH)` to a local variable (to generate it onc)